tests/test_script_queue.py

```diff
@@ -795,25 +814,43 @@ async def test_add_and_stop(self):
 
             await self.assert_next_queue(running=True, past_sal_indices=[I0])
 
+    @unittest.skipIf(
+        standardscripts is not None and externalscripts is not None,
+        "ts_standardscripts/ts_externalscripts successfully imported.",
+    )
+    async def test_bin_script_state_with_test_scripts(self):
```

Comment:
I gently suggest running this test regardless of whether the two ts script packages can be imported. It slows things down a bit, but it exercises some command-line arguments that would not otherwise be tested.

If you decide to do that then please tweak the new version history entry accordingly.